city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,174 @@
+from copy import deepcopy
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE, TENTATIVE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from parsel import Selector
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse_committee(self, respon, url):
```

Comment:
I'm a bit confused about this function. The committees and main minutes vs. the main meetings section shouldn't require much additional logic other than checking for text between parentheses for the committee name in `_parse_title`.

You could generate a `defaultdict` of items with tuples of the title and start date as keys and a list of links as the value (since those are the only things that will be different between places they're scraped), and use that to ensure uniqueness. It could work something like this.

```python
from collections import defaultdict

meeting_map = defaultdict(list)

for item in response.css("#content-238036 p, #content-232768 p"):
  title = self._parse_title(item)
  start = self._parse_start(item)
  if not start:
    continue
  meeting_map[(title, start)].extend(self._parse_links(item))

for key_tuple, links in meeting_map.items():
  title, start = key_tuple
  meeting = Meeting(
    title=title,
    start=start,
    links=links,
  )
  ...
```

Let me know if that makes sense, but something like that should replace this committee function

Reply:
Okay, implemented your ideas. I used the built in scrapy.Items class for the dictionary. The file is now almost a total rewrite, so would love comments when you have time. 
Thanks!